src/UserService.Broker/Consumers/GetUserInfoConsumer.cs

```diff
@@ -50,10 +47,17 @@ private object GetUserInfo(Guid userId)
                 throw new Exception(string.Join(", ", response.Message.Errors));
             }
 
-            var userPosition = response.Message.Body.UserPositionName;
+            var userPosition = response.Message.Body;
             var dbUser = repository.GetUserInfoById(userId);
 
-            return mapper.Map(dbUser, userPosition);
+            return new
+            {
+                UserId = dbUser.Id,
+                FirstName = dbUser.FirstName,
+                LastName = dbUser.LastName,
+                MiddleName = dbUser.MiddleName,
+                UserPosition = userPosition
+            };
```

Comment:
Hm, is it okay to map right here? Maybe there's another way to rework it...

Reply:
Do you have any suggestions? For example

Reply:
Do you have any suggestions? For example

Reply:
Maybe map it without UserPosition, and then set this field in the already existing DbUser